glean-core/src/upload/directory.rs

```diff
@@ -86,13 +95,22 @@ impl PingDirectoryManager {
     ///
     /// * `uuid` - The UUID of the ping file to be processed
     pub fn process_file(&self, uuid: &str) -> Option<PingRequest> {
-        let path = self.get_dir().join(uuid);
-        let file = match File::open(path) {
+        let path = match self.get_file_path(uuid) {
+            Some(path) => path,
+            None => {
```

Comment:
@badboy if you have a strong opinion on this, any chance we could codify it in some style guide and/or linter?